packages/flutter_tools/lib/src/android/gradle.dart

```diff
@@ -107,58 +106,6 @@ Iterable<String> _apkFilesFor(AndroidBuildInfo androidBuildInfo) {
   return <String>['app$flavorString-$buildType.apk'];
 }
 
-/// Returns true if the current version of the Gradle plugin is supported.
-bool _isSupportedVersion(AndroidProject project) {
```

Comment:
Since this only operates on AndroidProject, i've made it a method on AndroidProject